src/core/Table/Table.test.tsx

```diff
@@ -1672,3 +1673,520 @@ it('should change page size', () => {
   expect(rows[0].querySelector('.iui-cell')?.textContent).toEqual('Name1');
   expect(rows[49].querySelector('.iui-cell')?.textContent).toEqual('Name50');
 });
+
+it('should handle resize by increasing width of current column and decreasing the next ones', () => {
+  jest
+    .spyOn(HTMLElement.prototype, 'getBoundingClientRect')
+    .mockReturnValue({ width: 100 } as DOMRect);
+  const columns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+        },
+        {
+          id: 'description',
+          Header: 'description',
+          accessor: 'description',
+        },
+        {
+          id: 'view',
+          Header: 'view',
+          Cell: () => 'View',
+        },
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns,
+    isResizable: true,
+  });
+
+  const rows = container.querySelectorAll('.iui-table-body .iui-row');
+  expect(rows.length).toBe(3);
+
+  const resizer = container.querySelector('.iui-resizer') as HTMLDivElement;
+  expect(resizer).toBeTruthy();
```

Comment:
Maybe add test that there no `iui-resizer` when `isResizable` is false

Reply:
Added